packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+
+export interface ChipProps extends PropsWithChildren {
+  /** Accessible label, which can be different from the primary label. */
+  readonly ariaLabel?: string;
+  /** Disables both mouse and keyboard functionality, and updates the visual style of the Chip to appear disabled. */
+  readonly disabled?: boolean;
+  /** Adds more prominent text to act as a heading. Will be displayed on the left with a | separator. */
+  readonly heading?: string;
+  /** Changes Chip styling to inform the user of an issue. */
+  readonly invalid?: boolean;
+  /** The content of the chip. Will be displayed on the right if you include a heading. */
+  readonly label: string;
+  /** The accessible role the Chip is fulfilling. Defaults to 'button' */
+  readonly role?: string;
+  /** Used for accessibility purpopses, specifically using the tab key as navigation. Defaults to 0. */
+  readonly tabIndex?: number;
```

Comment:
For these defaults, you can use `@default 0` in a new line. You can search for "@default" in our codebase and you will find some examples. 

Here's one of them https://github.com/GetJobber/atlantis/blob/b1a3f289047063a7df19be0681c525145eb89d75/packages/components/src/InputGroup/InputGroup.tsx